spec/features/5_photos_spec.rb

```diff
@@ -1,120 +1,126 @@
 require "rails_helper"
 
-show_tests_in_browser = false
-do_not_show_tests_in_browser = false
+feature "Photos:" do
+
+  context "photos show page" do
+    before(:each) do
```

Comment:
Remember to avoid `before` and `let` in order to make it dead-simple for students to read tests in isolation:

https://teachersmanual.firstdraft.com/how-to-write-tests.html#avoid-before-let